### PR TITLE
Added SCRAPE_WANTED state. 

### PR DESCRIPTION
**Pull request by andrewconner**
Now, ACTIVE normalizedUris will not be scraped by default.

### CODE REVIEW

**Comment by andrewconner** [comment]
After this is rolled out, I will fix the old data in the db.

File: shoebox/app/com/keepit/model/NormalizedURI.scala [hunk 1]
```diff
@@ -119,8 +121,26 @@ class NormalizedURIRepoImpl @Inject() (
 
   override def save(uri: NormalizedURI)(implicit session: RWSession): NormalizedURI = {
     val saved = super.save(uri)
-    val scrapeRepo = inject[ScrapeInfoRepo]
-    scrapeRepo.getByUri(saved.id.get).getOrElse(scrapeRepo.save(ScrapeInfo(uriId = saved.id.get)))
+
+    lazy val scrapeRepo = scrapeRepoProvider.get
+    if(uri.state == NormalizedURIStates.INACTIVE || uri.state == NormalizedURIStates.ACTIVE) {
```
Comment (2is10) [inline shoebox/app/com/keepit/model/NormalizedURI.scala]: `if (`

Comment (andrewconner) [inline shoebox/app/com/keepit/model/NormalizedURI.scala]: :)

File: shoebox/app/com/keepit/model/NormalizedURI.scala [hunk 2]
```diff
@@ -119,8 +121,26 @@ class NormalizedURIRepoImpl @Inject() (
 
   override def save(uri: NormalizedURI)(implicit session: RWSession): NormalizedURI = {
     val saved = super.save(uri)
```
Comment (2is10) [inline shoebox/app/com/keepit/model/NormalizedURI.scala:123]: `save` ?

**Comment by 2is10** [comment]
Looks good to me.

**Pull request merged**